src/frontend/index.ts

```diff
@@ -330,6 +332,9 @@ export function createFlowrWindow(flowrStore: Store<IFlowrStore>): FlowrWindow {
       evt.sender.send('receiveScreenCapture', image.toDataURL())
     },
     openConfigMode: displayHiddenMenu,
+    getAudioDevicesPreferences: (evt: IpcMainEvent) => {
+      evt.sender.send('receiveAudioDevicesPreferences', flowrStore.get('audioDevices'))
```

Comment:
How the devices are stored in the json file config. There is something Automated that we can't see in this pr ?

Reply:
It's a manual operation to be done on the file system of the device. For this particular use case each device may have its own set of audio inputs and outputs, which means that this configuration can not be shared at the flowr organization level.
To share it across devices, we prepare an image that is remotely deployed.